starknet_py/net/account/account_client.py

```diff
@@ -428,8 +476,22 @@ async def create_account(
         :param signer: Signer used to create account and sign transaction
         :param chain: ChainId of the chain used to create the default signer
         :return: Instance of AccountClient which interacts with created account on given network
+
+        .. deprecated:: 0.4.7
+            This method has been deprecated and will be removed once transaction version 1 becomes default.
+            Compiled account contract will no longer be bundled with StarkNet.py and manaul account deployment
```

Comment:
Don't refer to it as a manual. We can just day to deploy an account of one's choice.